test/PerunPlutus/PerunSpec.hs

```diff
@@ -195,7 +195,6 @@ instance ContractModel PerunModel where
           PerunModel Nothing -> P.error "Update only works on existing channels"
           PerunModel (Just (_, tl, newFunding, newFunded)) -> PerunModel . Just $ (cs, tl, newFunding, newFunded)
       )
-  -- TODO add disputed bit to PerunModel and handle it here!
```

Comment:
Why is it fine not to do this here?

Reply:
I will refactor our `PerunModel`, I deleted it, because I thought the last parameter of our `PerunModel` was already associated with the `disputed` bit. This is not the case.

Reply:
I _lensified_ our `PerunModel` and added the `dispute` flag with extra tracking.